tap_github/repository_streams.py

```diff
@@ -1953,3 +1953,114 @@ def _write_schema_message(self):
         if not self._schema_emitted:
             super()._write_schema_message()
             self._schema_emitted = True
+
+
+class DependencyStream(GitHubGraphqlStream):
```

Comment:
Maybe use the plural? `DependenciesStream` just for consistency with *most* other streams (we've not actually been very consistent...)